pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -463,13 +501,81 @@ func (c *client) ModifyListeners(ctx context.Context, listenerArns []string, rou
 	}
 
 	for _, listener := range listenerArns {
-		if err := modifyListener(ctx, listener); err != nil {
-			return err
+		// Check if the current action type is forward
+		for _, rule := range describelistenerOutput.Rules {
+			for _, action := range rule.Actions {
+				if action.Type == elbtypes.ActionTypeEnumForward {
+					// Call modifyListenerRule only if action type is forward
+					if err := modifyListener(ctx, listener); err != nil {
+						return err
+					}
+				}
+			}
```

Comment:
This will delete all other rules of a given listener in case that listener contains rules of type `ActionTypeEnumForward` ? 🤔 So this code change means nothing, I guess. wdyt? @moko-poi

Reply:
@khanhtc1202 
Thank you for your review. 
I have made the necessary adjustments to the ModifyListeners and ModifyRules functions to address the concerns. Now, these functions only modify actions of type ActionTypeEnumForward and preserve all other action types, ensuring that existing listener and rule configurations are not inadvertently altered.